CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## v0.0.10 - in progress
+- Add expected_cells_count.
```

Comment:
The field name is `expected_cell_count` (singular).

Reply:
fixed. Thanks!